src/style.css

```diff
@@ -0,0 +1,40 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  background-image: url("./images/star.svg");
+
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars.stars--1 .stars__star:first-child {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars.stars--2 .stars__star:nth-child(-1n + 2) {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars.stars--3 .stars__star:nth-child(-1n + 3) {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars.stars--4 .stars__star:not(:last-child) {
+  background-image: url("./images/star-active.svg");
+}
+
+.stars.stars--5 .stars__star:nth-child(n) {
+  background-image: url("./images/star-active.svg");
+}
```

Comment:
Don't forger about checklist:
2. [CODE STYLE] - If several selectors MUST always have the same styles, group them using , to prevent accidental out ot sync in future

Also, you don't need to specify ".stars" selector every time and you can use Child combinator to avoid so many classes:
https://developer.mozilla.org/en-US/docs/Web/CSS/Child_combinator

Reply:
Thank you I made changes but I don't understand what difference btw ">" and "space"  for this problem
Why using > better than using space?